web/source/osk/visualKeyboard.ts

```diff
@@ -1031,11 +1031,6 @@ namespace com.keyman.osk {
         e.stopPropagation();
       }
 
-      // Do not attempt to support reselection of target key for overlapped keystrokes
-      if(e.touches.length > 1 || this.touchCount == 0) {
```

Comment:
So this is a control flow behaviour which worries me a little. Would it be better to throw a `cancelDelete()` before the return statement? Is there a chance that `e.touches.length` could be zero here leading to index out of range error further down?

Reply:
We're in touchMove here.  If the user moves their finger by just a pixel but remains on the backspace key, we don't want to cancel immediately... so we need some sort of check first.

Good point about 'could be zero', though it'd be a bit odd for a touchMove handler to have zero touches.

Reply:
Yeah I couldn't quite see a way for there to be zero touches :) Just trying to be real careful on what is a low-level change late in the game!

Reply:
I went ahead and added a zero-catch just in case.